### PR TITLE
[AR] Translation of 13-1.md

### PR DESCRIPTION
**Pull request by arahiche**

### CODE REVIEW

**Comment by JonathanSum** [comment]
Don't forget to put it in [AR ] Translation of 13-1.md

File: docs/ar/week13/13-1.md [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+<--!The formula above is to apply Laplacian to a function $h$ on the node $i$, which is the value of $h_i$ minus the mean value over its neighboring nodes $h_j$'s. Basically, if the signal is very smooth, the Laplacian value is small, and *vice versa*.
+
+### Step 2 : Fourier Functions
+
+The following is the eigen-decomposition of graph Laplacian,
+-->
+
+الصيغة أعلاه هي تطبيق لابلاسيان على الدالة $ h $ في العقدة $ i $ ، وهي قيمة $ h_i $ مطروحًا منها متوسط القيمة على العقد المجاورة لها ‪$‬h‪_‬j‪$‬. بشكل أساسي ، إذا كانت الإشارة سلسة جدًا ، تكون قيمة لابلاسيان صغيرة ، * والعكس صحيح *.
+
+### الخطوة الثانية: وظائف فوريي
```
Comment (AliEl-01) [inline docs/ar/week13/13-1.md:478]: i think term "fuctions" should be translated into "دوال"

File: docs/ar/week13/13-1.md [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+
+ما يلي هو التحليل الذاتي لمخطط لابلاسيان ،
```
Comment (AliEl-01) [inline docs/ar/week13/13-1.md:480]: النسبة تكون عبر ياء النسبة لتعطينا كلملة "مخطط لابلاسي" عوض مخطط لابلاسيان

and i think this is for every "Laplacian" term in the text!

**Review by AliEl-01 — CHANGES_REQUESTED** [review]

File: docs/ar/week13/13-1.md (hunk 2 above)
Comment (arahiche) [reply docs/ar/week13/13-1.md:480]: Laplacian denotes Laplace operator, a word that I preferred to write it as it in Arabic letters without translation, and I think لابلاسيان is more sounding than لابلاسي. However, ff you have a reference for لابلاسي so please let me know.

File: docs/ar/week13/13-1.md (hunk 1 above)
Comment (arahiche) [reply docs/ar/week13/13-1.md:478]: you're right

File: docs/ar/week13/13-1.md (hunk 2 above)
Comment (AliEl-01) [reply docs/ar/week13/13-1.md:486]: I think you already know that in arabic we have something we call it "النسبة" so the rule say that we add "ي" to the original term, in our case our original term is laplace=لابلاس so laplacian=لابلاسي.
I don't know if I make it clear or not but I think you already know that 😄

**Review by Atcold — APPROVED** [review]
I've updated `docs/_config.yml` for you.

**Pull request merged**